README.md

```diff
@@ -82,7 +82,14 @@ Pass additional parameters for for reporting. Optional.
 #### Example
 
 ```js
-cmp.init({ pubData: { browserId: 'gow59fnwohwmshz' }, isInUsa: false });
+cmp.init({
+    pubData: {
+        browserId: 'gow59fnwohwmshz',
```

Comment:
it would be useful to add a description of where these values could be acquired

Reply:
@sndrs Description added